.github/workflows/collect-stats.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: exit 1
 
   collect-stats-debug-cluster:
-    if: ${{ vars.ENABLE_DEBUG_CLUSTER == 'true' }}
+    if: ${{ vars.ENABLE_DEBUG_CLUSTER == 'true' && ! cancelled() }}
```

Comment:
The way Github has structured is kinda weird. Just adding `&& !cancelled()` keep them workflow running. But okay :)

Reply:
The job can finish with 3 outcomes: success, failure and cancelled. So `!cancelled()` just tells the runner to continue on any other outcome but 'cancelled'.

Reply:
Ooh makes sense.